src/components/sbb-radio-button-group/sbb-radio-button-group.spec.ts

```diff
@@ -1,21 +1,23 @@
-import { SbbRadioButtonGroup } from './sbb-radio-button-group';
-import { newSpecPage } from '@stencil/core/testing';
+import { expect, fixture } from '@open-wc/testing';
+import { html } from 'lit/static-html.js';
+import './sbb-radio-button-group';
 
 describe('sbb-radio-button-group', () => {
   it('renders', async () => {
-    const { root } = await newSpecPage({
-      components: [SbbRadioButtonGroup],
-      html: '<sbb-radio-button-group />',
-    });
+    const root = await fixture(html`<sbb-radio-button-group />`);
```

Comment:
In this case should be a problem, but better to be consistent
```suggestion
    const root = await fixture(html`<sbb-radio-button-group></sbb-radio-button-group>`);
```